packages/components/src/Banner/Banner.module.css

```diff
@@ -58,12 +80,17 @@
 .closeButton {
   margin: calc(var(--space-smallest) * -1) 0;
   align-self: flex-start;
+  mix-blend-mode: multiply;
 
   @media (--small-screens-and-below) {
     margin: calc(var(--space-small) * -1) 0;
   }
 }
 
+[data-theme="dark"] .closeButton {
+  mix-blend-mode: screen;
```

Comment:
keeps the dismiss button visible in dark mode while still blending the hover state with the status color

Reply:
![image](https://github.com/user-attachments/assets/b0707151-45c4-4ca5-97cb-a9b309875b18)